### PR TITLE
Looking for next step : ability for anybody to book anytime : Checking for more than 1 day

### PR DESCRIPTION
**Pull request by cvermot**
Adding an option to allow to check slots more than 1 day later.

### CODE REVIEW

**Comment by clemux** [comment]
I think it should be clearly stated that no booking appointment after the next day, at the time the script is run, will be booked, because, as of now, only "chronodoses" appointment motives are handled.

Unrelated, minor note: `timeWindow` should be `--time-window` in arguments, and `time_window` in the code.

**Comment by cvermot** [comment]
> I think it should be clearly stated that no booking appointment after the next day, at the time the script is run, will be booked, because, as of now, only "chronodoses" appointment motives are handled.

Not sure about that. Motive id 6970 seems to be for the Pfizer vaccine, and 7005 for the moderna one (if we select vaccines for 16/18 years old, we get only the 6970 motive, and if we select 50-55 years old with ARNm vaccines only, we get the also 6970 and 7005 motives).

> Unrelated, minor note: `timeWindow` should be `--time-window` in arguments, and `time_window` in the code.

Ok, done, checkingForMoreThan1Day is up to date.

**Comment by rbignon** [comment]
> I think it should be clearly stated that no booking appointment after the next day, at the time the script is run, will be booked, because, as of now, only "chronodoses" appointment motives are handled.

No, the script works well with that change (and that's what I do to test changes on doctoshotgun without waiting several hours).

The problem is that it encourages people to fraud :)

**Comment by cvermot** [comment]
> The problem is that it encourages people to fraud :)

For now, it could. We could wait the general opening (May 31) to push this PR in master, if you think people would cheat with this option.

File: README.md [hunk 1]
```diff
@@ -71,9 +72,24 @@ You can also give the patient id as argument:
...
+$ ./doctoshotgun.py paris roger.philibert@gmail.com -t 5
+Password:
+::: Looking for vaccine slots for Roger Philibert in 5 next (days)
+::: Trying to find a slot in Centre de Vaccination Covid 19 - Ville de Paris
+::: Best slot found: Mon May 17 11:00:00 2021
+::: Second shot: Fri Jun 25 15:20:00 2021
+::: Booking for Roger Philibert...
+::: Booking status: True
+::: Booked!
+```
```
Comment (rbignon) [inline README.md:104]: Output has changed

File: README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ Optional arguments:
 --center "<center_name>" [--center "<other_center>" …]  : filter centers to only choose one from the provided list
 --patient <index>                                       : select patient for which book a slot
 --debug                                                 : display debug information
+--time-window                                           : set how many next days the script look for slots
```
Comment (rbignon) [inline README.md]:
```suggestion
--time-window <days>                                    : set how many next days the script look for slots
```

**Comment by cvermot** [comment]
Merge done.

**Comment by cvermot** [comment]
Rebased.

**Pull request merged**